eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -49,6 +53,13 @@ jobs:
           node common/scripts/install-run-rush.js install
         displayName: "Install dependencies"
 
+      - template: eng/pipelines/templates/scripts/replace-relative-links.yml@azure-sdk-tools
+        parameters:
+          TargetFolder: $(Build.SourcesDirectory)
```

Comment:
This looks like it is going to update every md in the repo. Why are we not scoping?

Reply:
We want it to do that

Reply:
This is for the index generation

Reply:
Change target folder to ` $(Build.SourcesDirectory)\sdk\${{parameters.ServiceDirectory}}`

Reply:
done